### PR TITLE
dev-db/mariadb: add new versions

### PR DESCRIPTION
**Pull request by hydrapolic**
Copy of previous ebuilds, compiles on amd64.

### CODE REVIEW

File: dev-db/mariadb/Manifest [hunk 1]
```diff
@@ -1,7 +1,4 @@
...
-DIST mariadb-10.11.2.tar.gz 96372411 BLAKE2B 50ab57a1bb23817bebf992237055a2cbb8c946c325e29fef93b45dcb3be5212b8bdbf21d12eec15321712c431e0a9c3f924bd2b99d52cc829c12bbcc2513c6b7 SHA512 500e26461d36f16fb213b34db2795efb4e66668d443af97300495ea6978decaf3f49ad3f5a347822c6ff911c204a3e2263cb4351bc107aca0187b3474101c7b3
 DIST mariadb-10.11.3-patches-01.tar.xz 4984 BLAKE2B abc78c83505c745ab7320b99dcda8c667fed8332dede466142027b24957fa9ff3d03f8402679ce6cffdc89bc9f0559f8c717b7ec9df52a2c5db39195f096d731 SHA512 127386c9dc8ce242d9d309e31b9357a928ba816d24d94d60cbc8723fb3bd0fb136c7d049bfb37aac057f9ebcc73c155ac20b8bcf2542f89401140728c2dcf29a
-DIST mariadb-10.11.3.tar.gz 96761750 BLAKE2B 8cbce050faa3f1bd59115c2deadbacf3beff7fa4e0f9c3b78d338f8046e5e383919a9de90949574a5a72c1692dda9c1a47e32ab71030db5ea70dd6e951bd7822 SHA512 84de65230a99d7a0b2079cf1287723a30cb86e5c4f8e756503b260b8e495b454ed15fd5b959c9c12f6d8db2610993f3c8ff92cbdee6520bfbdcd5763dccdae07
```
Comment (thesamesam) [inline dev-db/mariadb/Manifest:4]: let's p.mask these too so people get a nice message saying they need to upgrade

Comment (hydrapolic) [reply dev-db/mariadb/Manifest:4]: Good idea Sam, p.mask added.

**Comment by thesamesam** [comment]
Thank you!

**Comment by hydrapolic** [comment]
Thank you @thesamesam!

**Pull request closed without merging**